src/components/fieldText/constants.ts

```diff
@@ -0,0 +1,68 @@
+import type { ArgTypes } from '@storybook/react';
```

Comment:
Please add a postfix to this file and update vite config to not include this to the package bundle as it relates to the storybook only.